tests/integration/mime/test_mime.py

```diff
@@ -28,48 +30,62 @@ def input_fn3():
         yield g
 
 
-def test_dummy_seg():
+def test_dummy_seg(mocker):
+    response_mock = mocker.Mock()
     f = Flow().add(uses='- !Buffer2URI | {mimetype: png}')
     with f:
-        f.index(input_fn=input_fn)
+        f.index(input_fn=input_fn, output_fn=response_mock)
 
+    response_mock.assert_called()
+    response_mock_2 = mocker.Mock()
     f = Flow().add(uses='- !Buffer2URI | {mimetype: png, base64: true}')
     with f:
-        f.index(input_fn=input_fn)
+        f.index(input_fn=input_fn, output_fn=response_mock_2)
```

Comment:
Can we `parametrise` this test by declaring pairwise [`inputfn`, `responsemock`] list at the top of the function?

Reply:
It is a sequential test, so is not either or, but one after the other.